### DISCUSSION TITLE
如何改变 BREAKING CHANGE文案

### DISCUSSION THREAD

**Discussion by huizhang001** [Zhengqbbb/cz-git#138] [General]
![image](https://github.com/Zhengqbbb/cz-git/assets/26496638/46c53b48-ddd7-4468-bb00-85ba095edc74)

**Comment by huizhang001** [comment]
![image](https://github.com/Zhengqbbb/cz-git/assets/26496638/dd1de109-179b-44c4-a567-d2a4a570627f)
- 首字符为 | 不换行
- 如何让重大变更描述，默认换行

**Comment by huizhang001** [comment]
![image](https://github.com/Zhengqbbb/cz-git/assets/26496638/16f6a579-758b-44dd-8dc2-874e6766af07)
 用了 "|" 换行符，下面会空一行，能不空行吗，想空行就两个”|“


![image](https://github.com/Zhengqbbb/cz-git/assets/26496638/c9b08cf7-9362-4f90-adb1-30984525f2e0)
感觉默认紧凑会好看一些

**Reply by huizhang001** [reply]
@Zhengqbbb 这个我知道了，由于 | 前后有空格，所以空出来一行，这个问题可以忽略

**Comment by Zhengqbbb** [comment]
cz-git｜czg 都是遵循 [conventionalcommits](https://www.conventionalcommits.org/en/v1.0.0/)
默认都是不换行的，首字母使用 | 下个版本会修
但是温馨提示，在一些 changelog 生成器中 BREAKING CHANGE 后如果没有信息（标题），即不生成

**Comment by huizhang001** [comment]
我想把BREAKING CHANGE 改成 中文版的  重大变更。。。目前有办法吗

**Reply by huizhang001** [reply]
@Zhengqbbb

**Reply by Zhengqbbb** [reply]
没有 不清楚你的需求是什么
但是一旦更改将无法正常使用一些 changelog 生成器

**Reply by huizhang001** [reply]
好，感谢~，我只是想让整个message信息以中文为主。
![image](https://github.com/Zhengqbbb/cz-git/assets/26496638/0a266daa-cfd3-4743-8a77-ee5cd07c0861)

**Reply by Zhengqbbb** [reply]
好的  我这些信息只是最好能提供给 changelog 生成器正则 parse，都是遵循 [conventionalcommits](https://www.conventionalcommits.org/en/v1.0.0/) 「标准 commit 信息」所以都是英文标识为主，如果你想要生成纯中文的 changelog 你可以在社区看看有没有自定义 changelog 生成器

**Reply by huizhang001** [reply]
czg 就很满足我的需求👍🏻
